game/scripts/vscripts/util/item.lua

```diff
@@ -77,3 +77,14 @@ function CDOTA_Item:SpendCharge(amount)
 		self:SetCurrentCharges(newCharges)
 	end
 end
+
+function CDOTA_Item:Copy(unit)
```

Comment:
This function is too specific, and usable only in one place. Don't see a reason to put it to a function. Also, there's `CopyItem` utility function.